monkey/tests/data_for_tests/agent_source_archive/src/vendor/mock_dependency.py

```diff
@@ -0,0 +1 @@
+__version__ = "1.0.0"
```

Comment:
This file is the same as `tests/data_for_tests/agent_plugin/src/vendor/mock_dependency.py`.

Reply:
See https://github.com/guardicore/monkey/pull/2736#discussion_r1053283984